Documentation/cmdref/cilium-agent.md

```diff
@@ -32,6 +32,7 @@ cilium-agent [flags]
       --bpf-ct-timeout-service-any duration           Timeout for service entries in non-TCP CT table (default 1m0s)
       --bpf-ct-timeout-service-tcp duration           Timeout for established service entries in TCP CT table (default 6h0m0s)
       --bpf-fragments-map-max int                     Maximum number of entries in fragments tracking map (default 8192)
+      --bpf-map-dynamic-size-ratio float              Ratio (0.0-1.0) of total system memory to use for dynamic sizing of CT, NAT and policy BPF maps. Set to 0.0 to disable dynamic BPF map sizing (default: 0.0)
```

Comment:
Does this conflicts with the `preallocate-bpf-maps` option?

Reply:
No, `--bpf-map-dynamic-size-ratio` only influences the number of entries in a map (like the `--bpf-*-map-max` flags) while `preallocate-bpf-maps` leads to the BPF hash maps being created without the `BPF_F_NO_PREALLOC` flag so they are preallocated by the kernel.